acknowledgments/index.md

```diff
@@ -45,3 +38,11 @@ Tj Vanderpoel sponsors a Tier 1 mirror in the USA.
 
 Pulux sponsors b-lej-de, which hosts [Void's monitoring
 infrastructure](https://grafana.s.voidlinux.org/).
+
+# Sponsor Emeritus
+
+## [Jürgen Buchmüller](#)
```

Comment:
I think we could keep the original link? Since https://pullmoll.github.io/ is hosted by GH pages, it will likely remain online for a long time.

Reply:
I'm not against it, but the site links to a paypal account that likely goes nowhere, and we shouldn't encourage people to find it...